java/integTest/src/test/java/glide/BatchTestUtilities.java

```diff
@@ -856,7 +857,7 @@ private static Object[] serverManagementCommands(BaseBatch<?> batch, boolean isA
                     OK, // configSet(Map.of("timeout", "1000"))
                     Map.of("timeout", "1000"), // configGet(new String[] {"timeout"})
                     OK, // configResetStat()
-                    "Redis ver. " + SERVER_VERSION + '\n', // lolwut(1)
+                    new VersionPatternMatcher(), // lolwut(1) - accepts both Redis and Valkey formats
```

Comment:
You can change it to
```suggestion
                    "ver. " + SERVER_VERSION + '\n', // lolwut(1)
```
Applicable to all clients

Reply:
Is ok to make it the check more simpler?